src/main/groovy/com/bmuschko/gradle/docker/tasks/AbstractDockerRemoteApiTask.groovy

```diff
@@ -62,6 +63,42 @@ abstract class AbstractDockerRemoteApiTask extends DefaultTask {
     @Optional
     final Property<String> apiVersion = project.objects.property(String)
 
+    /**
+     * Determines the timeout until a new connection is fully established.
+     *
+     * <b>Only used if HTTP is used as the transport.</b>
```

Comment:
Aren't we always using `ApacheDockerHttpClient` and therefore use the HTTP(s) protocol?

Reply:
I meant if we use a tcp connection URL - if a unix socket is used that timeout does not apply. I'll change that.